components/card/card.js

```diff
@@ -1 +1,27 @@
-export function createCharacterCard() {}
+export function createCharacterCard(
+  characterImage,
+  characterName,
+  characterStatus,
+  characterType,
+  characterOccurances
+) {
+  const card = document.createElement("li");
+  card.innerHTML = `<li class="card">
```

Comment:
because you created an element ("li") in 8, you dont need it in the line 9  (in .innerHtml)